.github/scripts/install.sh

```diff
@@ -2,8 +2,4 @@
 conda info -a
 
 conda create -q -n test-environment python=$PYTHON_VERSION numpy pyqt=4;
-source activate test-environment
-
-pip install --upgrade codecov
-pip install flake8
-pip install numpy
\ No newline at end of file
+source activate test-environment
```

Comment:
you still need the line
```bash
pip install -r requirements.txt -r requirements_dev.txt
```
after activating the environment in order to install all required packages